.rubocop.yml

```diff
@@ -1,3 +1,6 @@
 inherit_gem:
   rubocop-govuk:
     - config/default.yml
+
+Style/DoubleNegation:
```

Comment:
While I know this is a pain, there's not much point having a central style guide if we go and contradict it in each repo. Can we try two alternatives first:

- Disable this Cop for the issues we're not going to fix, so we still benefit from it for new code

- Fix the issues. They are all of the form `!!(regex =~ domainname)`, and can be fixed by doing e.g. `domainname&.match?(regex)`.

Reply:
Sure. I'll fix them. I mostly took one look at the code and wished for OctoDNS so we didn't have to maintain it!

Reply:
Done!